README.md

```diff
@@ -30,12 +30,42 @@ Currently, these distributions have existing packages for HyFetch:
 * Nix: `nix-env -i hyfetch` (Thanks to @ YisuiDenghua)
 * Guix: `guix install hyfetch` (Thanks to @ WammKD)
 
+### Method 3: Install using git and python
+
+Run the following commands:
+```sh
+git clone https://github.com/hykilpikonna/hyfetch.git
+cd hyfetch
+sudo python setup.py install
```

Comment:
Using `setup.py install` is not recommended because it cannot be uninstalled easily. Should use `pip install .` instead.

https://stackoverflow.com/questions/1550226/python-setup-py-uninstall